rpc_jobs/rpc_aio.yml

```diff
@@ -283,7 +319,9 @@
                 kibana_branch = env.STAGES.contains("Major Upgrade") ? env.UPGRADE_FROM_REF : env.KIBANA_SELENIUM_BRANCH
```

Comment:
We'll need to remember to update this assignment here when the `Minor Upgrade` is expanded to handle more stages.